src/zkasm_parser.jison

```diff
@@ -28,25 +28,38 @@ ROTL_C                  { return 'ROTL_C'; }
 HASHPOS                 { return 'HASHPOS'; }
 RID                     { return 'RID'; }
 MLOAD                   { return 'MLOAD' }
+F_MLOAD                 { return 'F_MLOAD' }
 MSTORE                  { return 'MSTORE' }
+MSTORE                  { return 'F_MSTORE' }
```

Comment:
This should be F_MSTORE

Reply:
It a typo, but need to check test, because with this mistake, a F_MSTORE generate an error because not recognized.

Reply:
remove F_MSTORE, no sense